packages/tic-tac-toe/app/controllers/games/index.ts

```diff
@@ -1,16 +1,34 @@
 import Controller from '@ember/controller';
 import {tracked} from '@glimmer/tracking';
 import {action} from '@ember/object';
+import {inject as service} from '@ember/service';
+import DS from 'ember-data';
+import UserService from '@statechannels/tic-tac-toe/services/user';
 
 export default class GamesIndexController extends Controller {
-  @tracked buyInAmount = 0.1;
+  @service store!: DS.Store;
+  @service user!: UserService;
+
+  @tracked buyInAmount = 0.001;
   @tracked showCreateGameModal = false;
 
   @action
-  protected createNewGame(event: Event): void {
+  protected async createNewGame(event: Event): Promise<void> {
```

Comment:
Why is this `async` now?

Reply:
I `await` the `newGame.save()` on line 31.